base_tier_validation/models/tier_validation.py

```diff
@@ -406,17 +409,19 @@ def _tier_validation_check_state_on_write(self, vals):
                         pending_reviews = reviews.filtered(
                             lambda r: r.status == "pending"
                         ).mapped("name")
+                        pending_reviews_str = "\n - ".join(pending_reviews)
                         raise ValidationError(
-                            _(
+                            self.env._(
                                 "This action needs to be validated for at least "
-                                "one record. Reviews pending:\n - %s "
-                                "\nPlease request a validation."
+                                "one record. Reviews pending:\n "
+                                "- %(pending_reviews_str)s "
+                                "\nPlease request a validation.",
+                                pending_reviews_str=pending_reviews_str,
```

Comment:
Same as previous comments. It's a shame to invalidate existing translatable terms just to use a kword argument on a term with only 1 arg